packages/calcite-components/src/demos/_assets/demo-theme.ts

```diff
@@ -41,24 +41,20 @@ export class DemoTheme extends HTMLElement {
     }
   }
 
-  updateTheme(newValue: string): void {
-    if (typeof newValue === "string") {
-      let tokensList;
-
-      try {
-        tokensList = JSON.parse(newValue);
-      } catch {
-        tokensList = newValue.split(",").map((t) => t.trim());
-      }
-
-      if (Array.isArray(tokensList)) {
-        const stringifiedTheme = setCSSVariables(tokensList);
-
-        if (this._el) {
-          this._el.style.cssText = stringifiedTheme;
-        } else {
-          this.setAttribute("style", stringifiedTheme);
-        }
+  updateTheme(tokens: string): void {
+    const tokensList = tokens
+      .split(",")
+      .map((t) => t.trim())
+      .filter(Boolean);
+
+    if (Array.isArray(tokensList)) {
+      console.log(tokensList);
```

Comment:
console.log("remove me"); 💻

Reply:
loooool, I need to install https://github.com/Esri/calcite-design-system/pull/11104 this week. Thanks for the catch and reminder! 🎣🧠